src/Illuminate/Mail/TransportManager.php

```diff
@@ -97,8 +97,8 @@ protected function createMailDriver()
      */
     protected function createMailgunDriver()
     {
-        $client = new HttpClient;
         $config = $this->app['config']->get('services.mailgun', []);
+        $client = new HttpClient(Arr::get($config, 'guzzleOptions', []));
```

Comment:
We also need to add this option as a placeholder in the laravel/laravel config/services.php That way you can kinda force the syntax.